pkgs/development/python-modules/grep-ast/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, python3
+, fetchFromGitHub
+}:
+
+python3.pkgs.buildPythonPackage rec {
+  pname = "grep-ast";
+  version = "3cf6200";
+
+  src = fetchFromGitHub {
+    owner = "paul-gauthier";
+    repo = "grep-ast";
+    rev = version;
+    sha256 = "sha256-xeHT8Zp1NyLeK2864DsFpNoxrct/EcUEbG+d33Gc/LQ=";
+  };
+
+  propagatedBuildInputs = with python3.pkgs; [
+    tree-sitter-languages
+    pathspec
+  ];
+
+  doCheck = false;
+
+  meta = with lib; {
+    description = "A tool to grep through the AST of a source file";
```

Comment:
```suggestion
    description = "Tool to grep through the AST of a source file";
```

is this a CLI?

Reply:
The package could be used as a CLI yes

Reply:
in that case this should probably also have `meta.mainProgram`